Source/Views/RepositoryCell/RepositoryCell.swift

```diff
@@ -51,6 +52,13 @@ struct RepositoryCell: View {
         } else {
             notifiedReposObservable.addRepository(viewModel)
         }
+        if let didChangeNotity = didChangeNotity, isNotifying {
+            didChangeNotity()
+            // Seems viewModel id is cached and status of button has been kept also
+            // So this is trick to make sure at RepositoryScreen, notify button is always notifying status
+            isNotifying.toggle()
```

Comment:
Can you provide the instruction to replicate this bug?

Reply:
Comment out that line @blyscuit 
Steps:
1. At Search repo screen, hit notify a few repos (A, B, C)
2. Back to Repo screen, hit unnotify repo A
3. Go to Search again, hit notify repo A again
4. Back to Repo screen and see the result

Current result:
- Repo A cell will display the button with `Notify` status not `Notifying`

Reply:
I think instead of declaring `@ObservedObject var notifiedReposObservable = NotifiedRepositoryObservable()` we should passing in closure to ViewModel to update `notifiedReposObservable` data. But that's a refactor so this is fine.

Reply:
I tried like you said last night. :lol: but that bug still happened